api/client/commands.go

```diff
@@ -392,6 +392,10 @@ func (cli *DockerCli) CmdLogin(args ...string) error {
 		return err
 	}
 	registry.SaveConfig(cli.configFile)
+
+	// encourage user to use `docker logout` in order to cleanup persisted login credentials
+	fmt.Fprintf(cli.out, "WARNING: login credentials persisted in %s If this is not a your personal machine, please `docker logout` to remove them when finished with the registry.\n", path.Join(os.Getenv("HOME"), registry.CONFIGFILE))
```

Comment:
@tiborvass now using homedir.Get()

Do we have anything like this already in the repository that would help with the formulation?